appendices/common-arguments.md

```diff
@@ -0,0 +1,135 @@
+# Common Arguments
+
+Here is the list and explanation of several arguments, which can be used with most commands.
+
+## if
+
+The `if` argument followed by a condition, defines whether a command will be executed or skipped. The condition is a C# macro and if the expression contains some spaces, you have to embrace it with `⊂⊃` special characters.
+
+| Argument Name | Argument Value | Command Execution |
+| ------------- | -------------- | ----------------- |
+| `if` | true | yes |
+| `if` | false | no |
+
+### Example 1
+
+```G1ANT
+♥x = 3
+program notepad if ♥x==3
+```
+
+In the example above, number 3 is assigned to the `♥x` variable.  This part of the script: `program notepad if ♥x==3` will only be executed, if the condition is true. In this case, it is, therefore G1ANT.Robot **will open** Notepad.
+
+### Example 2
+
+```G1ANT
+♥x = theater
+program notepad if ♥x=="movie"
+```
+
+In the example above, the word “theater” is assigned to the `♥x` variable.  The second line of the script — `program notepad if ♥x=="movie"` — will only be executed, if the condition is true. In this case it is false (`theater` is not equal to `movie`), therefore G1ANT.Robot **will not** open Notepad.
+
+Please note that the word `movie` has to be in double quotes, because this is how strings have to be declared in C#.
+
+### Example 3
+
+```G1ANT
+♥x = 3
+♥y = theater
+program notepad if ⊂♥x==3 && ♥y=="theatre"⊃
+```
+
+G1ANT.Robot **will open** Notepad.
```

Comment:
it won't open Notepad, because ♥y != "theatre"

Reply:
good point!